exchanges/zb/zb_wrapper.go

```diff
@@ -795,5 +792,77 @@ func (z *ZB) GetHistoricCandles(pair currency.Pair, a asset.Item, start, end tim
 
 // GetHistoricCandlesExtended returns candles between a time period for a set time interval
 func (z *ZB) GetHistoricCandlesExtended(p currency.Pair, a asset.Item, start, end time.Time, interval kline.Interval) (kline.Item, error) {
-	return z.GetHistoricCandles(p, a, start, end, interval)
+	ret, err := z.validateCandlesRequest(p, a, start, end, interval)
+	if err != nil {
+		return kline.Item{}, err
+	}
+
+	p, err = z.FormatExchangeCurrency(p, a)
+	if err != nil {
+		return kline.Item{}, err
+	}
+
+	startTime := start
+allKlines:
+	for {
+		klineParams := KlinesRequestParams{
+			Type:   z.FormatExchangeKlineInterval(interval),
+			Symbol: p.String(),
+			Since:  convert.UnixMillis(startTime),
+			Size:   int64(z.Features.Enabled.Kline.ResultLimit),
+		}
+
+		candles, err := z.GetSpotKline(klineParams)
+		if err != nil {
+			return kline.Item{}, err
+		}
+
+		for x := range candles.Data {
+			if candles.Data[x].KlineTime.Before(start) || candles.Data[x].KlineTime.After(end) {
+				continue
+			}
+			if startTime.Equal(candles.Data[x].KlineTime) {
+				// no new data has been sent
+				break allKlines
+			}
+			ret.Candles = append(ret.Candles, kline.Candle{
+				Time:   candles.Data[x].KlineTime,
+				Open:   candles.Data[x].Open,
+				High:   candles.Data[x].Close,
```

Comment:
High = Close

Reply:
Sweet catch! I did an audit and it occurs in a few locations! I updated all the I found in the latest commit